ui/lib/core/addon/components/form-field.hbs

```diff
@@ -16,22 +16,41 @@
   {{/unless}}
   {{#if @attr.options.possibleValues}}
     {{#if (eq @attr.options.editType "radio")}}
-      <div class="control columns" data-test-radio-input>
+      <div class="control {{unless this.isRadioGroup 'columns'}}" data-test-radio-input>
         {{#each (path-or-array @attr.options.possibleValues @model) as |val|}}
-          <div class="column is-narrow is-flex-center has-text-grey has-right-margin-s" data-test-input={{@attr.name}}>
+          <div
+            class="is-flex-center
+              {{unless
```

Comment:
Since there is a true and false value, i would use `{{if` instead of unless, it's easier to reason about. If we rename it to `this.radioColumns` it will make more semantic sense on line 19 as well